core/src/main/java/tachyon/command/TFsShell.java

```diff
@@ -470,6 +514,8 @@ public void printUsage() {
     System.out.println("       [pin <path>]");
     System.out.println("       [unpin <path>]");
     System.out.println("       [free <file path|folder path>]");
+    System.out.println("       [getUsedBytes <path>]");
+    System.out.println("       [getCapacityBytes <path>]");
```

Comment:
The path element is kind of confusing here, since the user may assume the command gives the info relative to a specific path.